src/rust/scheduler/scheduler.rs

```diff
@@ -199,13 +198,20 @@ impl Scheduler {
             }
             // There is some dropped task in this page, so iterate through it.
             if dropped != 0 {
+                let mut task_ids: RefMut<HashMap<u64, usize>> = self.task_ids.borrow_mut();
+
                 // Handle dropped tasks only.
                 for subpage_ix in BitIter::from(dropped) {
-                    if subpage_ix != 0 {
-                        let ix: usize = (page_ix << WAKER_BIT_LENGTH_SHIFT) + subpage_ix;
-                        tasks.remove(ix);
-                        pages[page_ix].clear(subpage_ix);
-                    }
+                    let index: usize = (page_ix << WAKER_BIT_LENGTH_SHIFT) + subpage_ix;
+                    println!("dropping task {:?}", index);
```

Comment:
If this is intentional, could you change it to a `trace!()` statement?

Reply:
sorry, that shouldn't be there.